src/cmd/nsfs.js

```diff
@@ -62,18 +62,19 @@ Options:
 
     ## single user mode
 
+    --simple <boolean>   (default false)        Starts a single user/fs mode
     --uid <uid>          (default as process)   Send requests to the Filesystem with uid.
     --gid <gid>          (default as process)   Send requests to the Filesystem with gid.
     --access_key <key>      (default none)      Authenticate incoming requests for this access key only (default is no auth).
     --secret_key <key>      (default none)      The secret key pair for the access key.
 
     ## multi user mode
 
-    --iam_json_schema                           Print the json schema of the identity files in iam dir.
-    --iam_ttl <seconds>     (default 60)        Identities expire after this amount of time, and re-read from the FS.
-    --config_root <dir>     (default none)      Configuration files for Noobaa standalon NSFS. It includes config files for environment variables(<config_root>/.env), 
-                                                local configuration(<config_root>/config-local.js), authentication (<config_root>/accounts/<access-key>.json) and 
-                                                bucket schema (<config_root>/buckets/<bucket-name>.json).
+    --iam_json_schema                                       Print the json schema of the identity files in iam dir.
+    --iam_ttl <seconds>     (default 60)                    Identities expire after this amount of time, and re-read from the FS.
+    --config_root <dir>     (default /etc/noobaa.conf.d)    Configuration files for Noobaa standalon NSFS. It includes config files for environment variables(<config_root>/.env), 
```

Comment:
I think we might need this
```suggestion
    --config_root <dir>     (default ${config.NSFS_NC_DEFAULT_CONF_DIR})    Configuration files for Noobaa standalon NSFS. It includes config files for environment variables(<config_root>/.env), 
```

Reply:
done